test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,11 +36,31 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() throws Exception {
+        // contains null
+        assertContainsNull(null, null);
+        assertContainsNull(null, "a", "b", null);
+
+        // does not contains null
+        assertNotContainsNull("abc", "", "abc", "ABC");
+        assertNotContainsNull("", "abc", "a", "abc");
+        assertNotContainsNull(1, Integer.valueOf(1));
```

Comment:
Good work on adding both positive and negative test cases. 
However do note that if the method accepts empty param, you should also test it to ensure that it meets your expected functionality (though doesn't really apply in this scenario, just a heads up 👍 )

Also, when adding test cases, think about how will the new test case will behave differently from the existing test cases.